_pages/business-units/18f/projects-partners/collaboration-tools.md

```diff
@@ -108,10 +108,12 @@ Here are some common barriers to using Drive with partners and some workarounds
 <details markdown="block">
 <summary>Trello alternatives</summary>
 
-- **Zenhub:** Not recommended from a compliance perspective. No FedRAMP in progress, no GSA IT approval for use of the service. From a user perspective, it's great that it works within GitHub and provides higher-level scoping and linking (e.g., epics and dependencies).
-- **GitHub Projects:** Not recommended from usability/UI perspective, as the interface is basic without the richness of other planning systems or the simplicity of Trello. Compliance/records considerations are the same as for GitHub.
-- **Jira:** The GSA IT Jira instance is not recommended. Like Zenhub, it integrates with GitHub and provides higher-level scoping and linking (e.g., epics and dependencies). It's pretty difficult to grant access to outsiders, and it's pretty heavyweight and difficult to configure.
-- **Microsoft Azure Devops Boards:** Bundled with Azure, but not within FedRAMP services in scope.
+- [**Airtable**]({{site.baseurl}}/tools/airtable/)
+- [**GitHub Project Boards:**](https://docs.github.com/en/github/managing-your-work-on-github/managing-project-boards) Not recommended from usability/UI perspective, as the interface is basic without the richness of other planning systems or the simplicity of Trello. Compliance/records considerations are the same as for [GitHub]({{site.baseurl}}/github/).
+- **Jira:** The GSA IT Jira instance is not recommended. Like ZenHub, it integrates with GitHub and provides higher-level scoping and linking (e.g., epics and dependencies). It's pretty difficult to grant access to outsiders, and it's pretty heavyweight and difficult to configure. Contact <gsa-jira.support@gsa.gov> for more information.
+- **Microsoft Azure Devops Boards:** Bundled with Azure, but not within [FedRAMP services in scope](https://docs.microsoft.com/en-us/azure/azure-government/compliance/azure-services-in-fedramp-auditscope).
+- [**Smartsheet**]({{site.baseurl}}/tools/smartsheet/)
+- [**ZenHub:**](https://www.zenhub.com/) Works within GitHub and provides higher-level scoping and linking (e.g., epics and dependencies). Unfortunately, [we cannot expand use at GSA](https://github.com/18F/tts-tech-portfolio/issues/1040).
```

Comment:
Since I alphabetized these, may be easier to review individual commits to see the changes.